neard/src/lib.rs

```diff
@@ -193,6 +193,11 @@ pub fn apply_store_migrations(path: &String, near_config: &NearConfig) {
         // version 17 => 18: add `hash` to `BlockInfo` and ColHeaderHashesByHeight
         migrate_17_to_18(&path);
     }
+    if db_version <= 18 {
+        info!(target: "near", "Migrate DB from version 18 to 19");
+        // version 18 => 19: populate ColEpochValidatorInfo for archival nodes
+        migrate_18_to_19(&path, near_config);
```

Comment:
Hm, something odd jumps at me. Here we migrate 18 to 19, but in the next if we migrate forward from `18`. Naively, I'd expect to see `migrate_18_to_rectify_inflation` and friends first, and `migrate_18_to_19` last. Not sure if it's just unclear naming or a real problem.

Reply:
yeah. For nightly feature we need to change the name of the migration function when we stabilize them. Please feel free to suggest better ways to name those functions.

Reply:
Nightly are conditional migrations, and at the time of writing we implement a migration path from version X (e.g. 18) to the state that nightly wants, so it seems fine to me to name them `migrate_18_to_nightly_feature`. If it ever gets in conflict, we need to update the migration and the function name, but otherwise, we can keep it and indicate that it was initially implemented on top of db version 18. Not perfect, but it seems to do the trick for us.

Thugh, explicit `db_version <= 18` checks seem weird to me. Should it be `db_version < 10000`?